cfme/tests/intelligence/reports/test_validate_chargeback_report.py

```diff
@@ -27,8 +28,11 @@
     pytest.mark.tier(2),
     pytest.mark.meta(blockers=[BZ(1433984, forced_streams=["5.7", "5.8", "upstream"]),
                                BZ(1468729, forced_streams=["5.9"]),
+                               BZ(1531600, forced_streams=["5.9"]),
                                BZ(1511099, forced_streams=["5.7", "5.8"],
-                                  unblock=lambda provider: not provider.one_of(GCEProvider))]),
+                                  unblock=lambda provider: not provider.one_of(GCEProvider)),
+                               BZ(1531554, forced_streams=["5.8"],
```

Comment:
Looks like this bug is relevant for infra providers as well, as the UI element is missing in PRT screenshots for rhv41, vsphere*, etc in 5.8 results.